cg/models/scout/scout_load_config.py

```diff
@@ -61,14 +61,21 @@ class ScoutMipIndividual(ScoutIndividual):
     mitodel_file: Optional[str] = None
 
 
-class ScoutBalsamicIndividual(ScoutIndividual):
+class ScoutCancerIndividual(ScoutIndividual):
     tumor_type: Optional[str] = None
     tmb: Optional[str] = None
     msi: Optional[str] = None
     tumor_purity: float = 0
+
+
+class ScoutBalsamicIndividual(ScoutCancerIndividual):
     vcf2cytosure: Optional[str] = None
 
 
+class ScoutRnafusionIndividual(ScoutCancerIndividual):
+    pass
```

Comment:
Feels odd to me to have an empty class and one with a field that is optional. Is there a reason to not just use `ScoutCancerIndividual`? I don't have a strong opinion though, so feel free to keep it as it is if you think it's a better idea :)
 
```suggestion
class ScoutCancerIndividual(ScoutIndividual):
    tumor_type: Optional[str] = None
    tmb: Optional[str] = None
    msi: Optional[str] = None
    tumor_purity: float = 0
    vcf2cytosure: Optional[str] = None    
```

Reply:
I agree but I felt it was good to use the same schema but maybe it doesn't matter. I'll remove it